src/components/ToolTip.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+defineProps({
+    content: {
+        type: String,
+        required: true,
+    },
+});
+</script>
+
+<template>
+<q-tooltip class="tooltip">
+    <pre class="tooltip__content">{{ content }}</pre>
+</q-tooltip>
+</template>
+
+<style lang="sass" scoped>
+.tooltip
+    word-break: break-all
+    max-height: 100% !important
+    &_content
```

Comment:
This rule will not apply to the class `tooltip__content`, correct?
Also, it is scoped and uses SASS. Maybe, it's worth rewriting the entire style section.

```suggestion
    &__content
```

Reply:
youre right, i will rewrite to match our standards 👍